packages/@ember/object/type-tests/computed.test.ts

```diff
@@ -0,0 +1,28 @@
+import { computed } from '@ember/object';
```

Comment:
Could probably add an `expectTypeOf` here to check for the return type of the various forms:

```ts
expectTypeOf(computed('foo')).toEqualTypeOf<PropertyDecorator>();
```

etc.—and the same for the CP macro type tests below.

Reply:
This is partially completed.